.gitignore

```diff
@@ -14,3 +14,5 @@ AppPackages
 Microsoft.IdentityModel.Clients.ActiveDirectory.XML
 *.htm
 .nugetPackageRoot/
+/devApps/AdalAndroidTestApp/AdalAndroidTestApp.csproj.bak
+/ADAL.NET.NoWinRT.sln.DotSettings.user
```

Comment:
you probably want to remove this DotSettings.user file as well (from the commit)

Reply:
I added both to .gitignore to ensure those files are not committed